utils.ts

```diff
@@ -61,7 +61,7 @@ export const fail = (name: string, message: string) => {
   return quit(err)
 }
 
-export const run = async (command: string, args: string[]) => {
+export const run = (command: string, args: string[], errMsg?: string) => {
```

Comment:
Nice refactor. I originally had a `run` that was async, but never got back to changing this. I like the error message being passed in also.

Reply:
yeah, I saw that so I figured this was safe